src/js/organisms/filter-form.js

```diff
@@ -1,4 +1,4 @@
-const { parseForm, parseQueryParams } = require('../utils');
```

Comment:
This file might be better reviewed by viewing just the current state, not the diff